### PR TITLE
- Add htmldate library to extract date

### PR DESCRIPTION
**Pull request by huyqd**
- dynamic DAGs

### CODE REVIEW

File: dags/DAG_url_processor.py [hunk 1]
```diff
@@ -36,7 +37,7 @@ def extract_data(url):
         article.parse()
 
         return {
-            'published_at': article.publish_date,
+            'published_at': article.publish_date or find_date(url),
```
Comment (nikolasrieble) [inline dags/DAG_url_processor.py:40]: If the publish_date is None, find_date(url) will be set, right? 
But what if the publish_date is empty i.e. "" and not None

File: dags/DAG_url_processor.py [hunk 2]
```diff
@@ -52,15 +53,24 @@ def extract_data(url):
...
-          catchup=False,
-          )
-
-with dag:
-    processor = PythonOperator(task_id='de_url_processor_operator',
-                               python_callable=url_processor,
-                               op_kwargs={'language': 'de'})
+
+LANGUAGES = ['de', 'tr']
+for language in LANGUAGES:
```
Comment (nikolasrieble) [inline dags/DAG_url_processor.py:68]: This is almost identical to the previous setup. Note that both DAGs should have a starting date that is different by 30 seconds such that they are not scheduled at the same time.

Comment (huyqd) [reply dags/DAG_url_processor.py]: Updated

File: dags/DAG_url_processor.py (hunk 1 above)
Comment (huyqd) [reply dags/DAG_url_processor.py]: Updated. Could you check?

**Comment by huyqd** [comment]
Adapt your suggestion @nikolasrieble

**Review by nikolasrieble — APPROVED** [review]
Looks good, please test before merging.

**Pull request merged**